proto/replicationdata.proto

```diff
@@ -21,20 +21,23 @@ option go_package = "vitess.io/vitess/go/vt/proto/replicationdata";
 
 package replicationdata;
 
-// Status is the replication status for MySQL (returned by 'show slave status'
-// and parsed into a Position and fields).
+// Status is the replication status for MySQL/MariaDB/File-based. Returned by a
+// flavor-specific command and parsed into a Position and fields.
 message Status {
   string position = 1;
+  // deprecated fields
+  reserved 2;
+  reserved 3;
+  uint32 seconds_behind_master = 4;
+  string master_host = 5;
+  int32 master_port = 6;
+  int32 master_connect_retry = 7;
   // RelayLogPosition will be empty for flavors that do not support returning the full GTIDSet from the relay log, such as MariaDB.
   string relay_log_position = 8;
   string file_position = 9;
   string file_relay_log_position = 10;
   uint32 master_server_id = 11;
-  bool slave_io_running = 2;
-  bool slave_sql_running = 3;
-  uint32 seconds_behind_master = 4;
-  string master_host = 5;
-  int32 master_port = 6;
-  int32 master_connect_retry = 7;
   string master_uuid = 12;
+  bool io_thread_running = 13;
+  bool sql_thread_running = 14;
```

Comment:
Is this because it's a new feature? If so, how come the ids are this low? Were they unused before?

Reply:
This is renaming slave_io_running and slave_sql_running. The old names have been removed and marked as reserved, and replaced with new names.

Reply:
the extra diff is because the fields were not ordered by number and it was confusing.

Reply:
I was asking about why you didn't continue to use the original numbers 2 & 3, like you did for the other cases.

Reply:
can I do that? I didn't think you could rename proto fields. I looked at the rest of the diff and the only other proto change is force_start_slave -> force_start_replication where also I reserved 3 and added 5.

Reply:
from offline discussion, it seems like the correct approach is to rename the fields. reserving and creating new fields won't work across versions.